main.tf

```diff
@@ -170,6 +170,16 @@ module "application_gateway" {
   gateway_allowed_address_prefix   = var.inbound_allowed_address_prefix
   gateway_allowed_address_prefixes = var.inbound_allowed_address_prefixes
 
+  # If a single node is deployed the correct backend port is 7200, for cluster is 7201
+  gateway_backend_port = var.node_count != 1 ? var.gateway_backend_port : 7200
+
+  # Health probe settings
+  gateway_probe_path      = var.node_count != 1 ? var.gateway_probe_path : "/protocol"
```

Comment:
My suggestion here is:

- Remove `/rest/cluster/node/status` as a default value and move it here. Same for the port.
- Use var.gateway_probe_path as an override when specified. Same for the port.
- Maybe move these inside the submodule? I kinda prefer such decisions to be there.